lib/network/kmip/transport/TransportTemplate.js

```diff
@@ -0,0 +1,154 @@
+'use strict'; // eslint-disable-line
+
+const assert = require('assert');
+
+const DEFAULT_PIPELINE_DEPTH = 8;
+const DEFAULT_KMIP_PORT = 5696;
+
+class TransportTemplate {
+    /**
+     * Construct a new object of the TransportTemplate class
+     * @param {Object} channel - Typically the tls object
+     * @param {Object} options - Instance options
+     * @param {Number} options.pipelineDepth - depth of the pipeline
+     * @param {Object} options.tls - Standard TLS socket initialization
+     *                               parameters
+     * @param {Number} options.tls.port - TLS server port to connect to
+     */
+    constructor(channel, options) {
+        this.channel = channel;
+        this.options = options;
+        this.pipelineDepth = Math.max(1, options.pipelineDepth ||
```

Comment:
Just want to confirm that the 0 case is intended to go to the default rather than the max of 1. For example:

* if `options.pipelineDepth` is < 0 `this.pipelineDepth` is 1
* if `options.pipelineDepth` is equal to 0 `this.pipelineDepth` is 8

Reply:
exact! (actually, 1 is the min)